_posts/2025-03-05-next-scala-lts-jdk.md.md

```diff
@@ -0,0 +1,30 @@
+---
+category: announcement
+permalink: /news/next-scala-lts-jdk.html
+title: "JDK 17 will be the next minimum version required by Scala 3"
+by: Tomasz Godzik, VirtusLab & Scala Core Team
+---
+
+We recently asked for community input about changing the JDK version used by the
+Scala 3 compiler. We wanted to thank everyone for their input and announce the
+decision to switch to JDK 17 as the new minimum, starting with Scala 3.8 minor
+and the next LTS which will mostly likely be Scala 3.9.
```

Comment:
```suggestion
and the subsequent new LTS which will mostly likely be Scala 3.9.
```
given we have release series called `Next` and `LTS`, using  them in a single sentence in a different meaning sounds off.